docker-compose.dev.yml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+services:
+  app:
+    build: .
+    depends_on:
+      - postgres
+    environment:
+      NODE_ENV: development
+      DATABASE-URL: ${TEST-DB}
```

Comment:
nice work. howerver, it is `TEST_DB` not `TEST-DB` please check the env.sample, also check on `DATABASE_URL`